packet/flatcar-linux/kubernetes/workers/cl/install.yaml.tmpl

```diff
@@ -88,14 +88,57 @@ storage:
             mkfs.ext4 /dev/md/node-local-storage
           }
 
+          function create_disk_specific_data_raid() {
+            # Ignore the disk on which Linux is installed when selecting disks for RAID.
+            local osdisk="$1"
+            local major_numbers="$2"
+            # for hdd value is 1 and for ssd value is 0
+            local disk_type="$3"
+            # RAID device path on disk
+            local device_path="$4"
+            local setup_fs_on_raid="$5"
+
+            # Select disks for RAID.
+            local disks=$(lsblk -lnpd -o name,rota -I "$${major_numbers}" \
+              | grep "$${disk_type}" \
```

Comment:
As stated in https://github.com/kinvolk/lokomotive-kubernetes/pull/54, this is buggy because grep for 0 or 1 will also match the names `sda1` and `nvme0n1`.

Reply:
As simple workaround adding a space would work: `grep " $${disk_type}"`

Reply:
Fix in https://github.com/kinvolk/lokomotive-kubernetes/pull/54